src/assets/content/privacy-policy.md

```diff
@@ -40,7 +40,9 @@ Using [your own node](https://support.mycrypto.com/developers/run-your-own-node-
 
 Some MyCrypto products have server-side functions that are hosted by a third-party provider and employ third-party security and functionality services. While MyCrypto and its third-party providers may collect limited information as is standard and appropriate to secure our Services and provide the best user experience, no personally identifiable information content associated with such information is stored. For example, we do not store information in a way that can track a private key, public address, or inputted transaction to a specific IP address or username.
 
-Some MyCrypto products collect minimal usage analytics on the client side in order to better understand how people are using the product. We use an open-source analytics platform called [Matomo](https://matomo.org/why-matomo/) (previously Piwik), which we host on our own servers and have customized extensively to limit the information gathered. This ensures no third party, including Matomo, has access to your data. We protect your information using the latest technology and best practices for technical and administrative security. This data is not personally identifiable and is collected solely to improve our products.
+Our main product offering ([MyCrypto App](https://app.mycrypto.com)) makes use of [Segment](https://segment.com/) for a data warehouse for anonymised analytics data. We collect the data grouped by the user using a randomly generated identification string. You can disable this and refresh your unique ID string whenever you like in your [app settings](https://app.mycrypto.com/settings) (see "Product Analytics"). We route the data collection requests through our own instance of [blockstack/stats](https://github.com/MyCryptoHQ/stats).
```

Comment:
for a data warehouse for anonymised analytics data. -> as a data warehouse for anonymized analytics data.